sync/src/main/kotlin/gropius/sync/AbstractSync.kt

```diff
@@ -107,7 +107,7 @@ abstract class AbstractSync(
      * @return Conversion information
      */
     abstract suspend fun syncComment(
-        imsProject: IMSProject, issueId: String, issueComment: IssueComment
+        imsProject: IMSProject, issueId: String, issueComment: IssueComment, users: List<User>
```

Comment:
sooo, technically you could  copy paste a users doc in all applicable methods in this file

Reply:
195827e3be0eb2161f4648cee5e36ca93118f497